toolchain/driver/compile_subcommand.cpp

```diff
@@ -109,6 +112,25 @@ object output can be forced by enabling `--force-obj-output`.
       },
       [&](auto& arg_b) { arg_b.Set(&output_filename); });
 
+  b.AddOneOfOption(
+      {
+          .name = "opt",
+          .help = R"""(
+Selects the amount of optimization to perform.
+)""",
+      },
+      [&](auto& arg_b) {
+        arg_b.SetOneOf(
+            {
+                // We intentionally don't expose O2 and Os.
+                arg_b.OneOfValue("none", llvm::OptimizationLevel::O0),
+                arg_b.OneOfValue("debug", llvm::OptimizationLevel::O1),
+                arg_b.OneOfValue("speed", llvm::OptimizationLevel::O3),
+                arg_b.OneOfValue("size", llvm::OptimizationLevel::Oz),
```

Comment:
Given that LLVM calls this `min_size`, what do you think about that name? I could go either way.